### ISSUE TITLE
Windows Link does not work

### ISSUE THREAD

**Issue by mirisbowring** [Maqsim/stackoverflow-app#3]
Hi,

#1 mentioned that the download links are not available (a year ago).  
There was a solution for MacOS.  
  
I am using windows, so i cannot just execute npm install in the repo.  
  
I appreciate every possible solution!

Regards

**Comment by Maqsim** [comment]
Hi, sorry for very (VERY) long time offline. It was a hobby project. Do you want to use the app?

**Comment by encryptedtouhid** [comment]
> Hi, sorry for very (VERY) long time offline. It was a hobby project. Do you want to use the app?

hell yeah, man. I am following this issue for a long time. don't close this. please complete this feature. i am here for your support.

**Comment by Maqsim** [comment]
@encryptedtouhid Hey dude. I am soo busy right now and this project became old. But I have an idea for near future, make YT videos and make Stackoverflow App from scratch using modern tech-stack. Do you think that is a good idea?

**Comment by encryptedtouhid** [comment]
> @encryptedtouhid Hey dude. I am soo busy right now and this project became old. But I have an idea for near future, make YT videos and make Stackoverflow App from scratch using modern tech-stack. Do you think that is a good idea?

Yes of course that it would be great. looking forward to it. please keep me in touch.

**Comment by Maqsim** [comment]
@encryptedtouhid Cool. Probably will start making in a month or so when some projects are finished

**Comment by encryptedtouhid** [comment]
> @encryptedtouhid Cool. Probably will start making in a month or so when some projects are finished

awesome. looking forward to it man.

**Comment by Maqsim** [comment]
Hey @encryptedtouhid I have continued work on the project and now it's on new Electron (with React + Typescript). Stay tune for updates.
